src/TheClassLessTheme/Views/Layout.liquid

```diff
@@ -35,9 +37,7 @@
         <main>
             <div id="top" class="page" role="document">
             {% render_section "Navigation", required: false %}
-                <main role="main">
```

Comment:
Miért törölted ki? Nem elég lett volna csak a `role`-t leszedni róla?

Reply:
Nem, mert felette van még egy <main>, és arra is van rule hogy csak egy lehet.

Reply:
Ah tényleg.